users/userRoutes.js

```diff
@@ -0,0 +1,96 @@
+
+const express = require('express');
+const router = express.Router();
+const db = require('./userDb');
+
+function allCap(req, res, next){
+    req.body.name = req.body.name.toUpperCase();
+    next();
+}
+
+
+router.get('/', async (req, res)=>{
+    
+    const users = await db.get();
+    try{
+        res.status(200).json(users)
+    }
+    catch(er){
+        res.status(500).json({message: 'There was an error retrieving the data'})
+    }
+
+});
+
+router.get('/:id', async (req, res)=>{
+
+ let {id} = req.params;
+ let user = await db.get(id);
+
+ try{
+    res.status(200).json(user)
```

Comment:
you need to handle the instance where the id doesn't exist. Right now it replies with a 200 OK status message, which isn't what you want in that case.